metrics/metrics_test.go

```diff
@@ -1,13 +1,20 @@
-package metrics_test
+package metrics
 
 import (
 	"testing"
-
-	"github.com/coopnorge/go-datadog-lib/v2/metrics"
 )
```

Comment:
Please revert, ref https://github.com/coopnorge/go-datadog-lib/pull/524/files#r2040069549

```suggestion
import (
	"testing"

	"github.com/coopnorge/go-datadog-lib/v2/metrics"
)
```